src/plugins/plugin.legend.js

```diff
@@ -219,6 +219,26 @@ export class Legend extends Element {
     return totalWidth;
   }
 
+  /**
+   * @private
+   */
+  _calculateItemSize(boxWidth, labelFont, ctx, legendItem, _itemHeight) {
+    const itemWidth = boxWidth + (labelFont.size / 2) + ctx.measureText(legendItem.text).width;
+    const itemHeight = this._calculateItemHeight(_itemHeight, legendItem, labelFont.lineHeight);
+    return {itemWidth, itemHeight};
+  }
+
+  /**
+   * @private
+   */
+  _calculateItemHeight(_itemHeight, legendItem, fontLineHeight) {
```

Comment:
Looks like all of these new helpers could be placed in the module scope (and thus would no need to be marked private as they would not be visible anyway)

I'd prefer it that way for minification and cleaner prototype.